README.md

```diff
@@ -27,137 +71,182 @@ LibreSplit requires the following dependencies on your system to compile:
 - `libjansson`
 - `luajit`
 
-## Installation
+Install the required dependencies:
+
+<details>
+<summary>Debian-based systems</summary>
 
-### Package manager (recommended)
-* AUR/Arch based distro: [libresplit-git](https://aur.archlinux.org/packages/libresplit-git)
+```sh
+sudo apt update
+sudo apt install build-essential libgtk3-dev libjansson-dev meson luajit
+```
+</details>
+
+<details>
+<summary>Arch-based systems</summary>
+
+```sh
+sudo pacman -Syu
+sudo pacman -S gtk3 jansson luajit git meson
+```
+</details>
 
-### Manual install
-```bash
-git clone https://github.com/wins1ey/LibreSplit
+Clone the project:
+
+```sh
+git clone https://github.com/LibreSplit/LibreSplit
 cd LibreSplit
+```
+
+Now compile and install:
+
+```sh
 meson setup build -Dbuildtype=release
 meson compile -C build
 meson install -C build
 ```
 
-## Getting Started
+All done! Now you can start the desktop **LibreSplit** or run `/usr/local/bin/libresplit`.
+
+---
+
+## Usage
+
+When you start **LibreSplit** for the first time, it will create the `libresplit` directory in your config directory (it will usually be `~/.config/livesplit`). Such directory will contain:
```

Comment:
```suggestion
When you start **LibreSplit** for the first time, it will create the `libresplit` directory in your config directory (it will usually be `~/.config/libresplit`). Such directory will contain:
```